proxy/contracts/predeployed/TokenManager.sol

```diff
@@ -400,8 +413,18 @@ contract TokenManager is PermissionsForSchain {
         );
     }
 
-    // Receive money from main net and Schain
-
+    /**
+     * @dev Allows MessageProxy to post operational message to mainnet
+     * or SKALE chains. Also used to refill mainnet SKALE chain agent to allow SKALE chain users 
```

Comment:
I think it is a copy of DepositBox contract. "postMessage" function could not refill main net SKALE chain agent

Reply:
Clarified.